script.js

```diff
@@ -1 +1,266 @@
+                break;
+            case 'tanh':
+                if(btnDeg.classList.contains("active"))
+                    display.value = mnjs.tanh.deg(+display.value);
+                else 
+                    display.value = Math.tanh(display.value);
+                break;
+            case 'floor':
+                display.value = Math.floor(display.value);
+                break;
+            case 'ceil':
+                display.value = Math.ceil(display.value);
+                break;
+            case 'round':
+                display.value = Math.round(display.value);
+                break;
+            case 'sign':
+                display.value = Math.sign(display.value);
+                break;
+            case 'trunc':
+                display.value = Math.trunc(display.value);
+                break;
+            case 'log':
+                display.value = Math.log10(display.value);
+                break;
+            case 'ln':
+                display.value = Math.log(display.value); 
+                break;
+            case 'square':
+                display.value = Math.pow(display.value,2);
+                break;   
+            case '1/x':
+                display.value = Math.pow(display.value,-1);
+                break; 
+            case '|x|':
+                display.value = Math.abs(display.value);
+                break; 
+            case 'exp':
+                display.value = Math.exp(display.value);
+                break;  
+            case 'sqrt':
+                display.value = Math.sqrt(display.value);
+                break;
+            case 'fact':
+                try {
+                    display.value = func_fact();
+                  } catch (err) {
+                    display.value = err.message;
+                  }
+            break; 
+            case '10raisex':
+                display.value = Math.pow(10,display.value);
+                break;
+            case '+/-':
+                display.value = display.value*(-1);
+                break;
+            case 'mc':
+                func_mc();
+                break;
+            case 'mr':
+                func_mr(); 
+                break;
+            case 'mplus':
+                func_mplus();
+                break;
+            case 'mminus':
+                func_mminus();
+                break;
+            case 'ms':
+                func_ms();
+                break;  
+            case '2nd':
+                func_2nd();
+                break;
+            case 'cube':
+                display.value = Math.pow(display.value,3);
+                break;
+            case '2raisex':
+                display.value = Math.pow(2,display.value);
+                break;      
+            case 'cbrt':
+                display.value = Math.cbrt(display.value);
+                break;
+            case 'eraisex':
+                display.value = Math.pow(2.71828182846,display.value);
```

Comment:
Not necessary, but you can use **Math.E** instead of typing value of _e_ manually.